theme/static/css/nwcpp.css

```diff
@@ -16,17 +16,19 @@
 }
 
 .content-wrapper {
-	width: 1100px;
+	width: 1200px;
 	margin: 0 auto;
 }
 
 .sidebar {
```

Comment:
So it looks like we have two sidebars now.  The new sidebar looks nice.  It does have the side effect of needing a wider browser window.  Lloyd, are we OK with that?

Also, the sidebar will need a bit of cleanup.  For instance, Google+ is no longer a thing.  I can do that in another changelist.